rust/routee-compass/src/app/compass/compass_app.rs

```diff
@@ -258,7 +258,10 @@ impl CompassApp {
         &self,
         query: serde_json::Value,
     ) -> Result<Vec<serde_json::Value>, CompassAppError> {
-        let search_result = self.search_app.run_vertex_oriented(&query);
+        let search_result = self
+            .search_app
+            .run_vertex_oriented(&query)
+            .or_else(|_| self.search_app.run_edge_oriented(&query));
```

Comment:
sure, i think it would be a tad less ambiguous in rare circumstances. like we talked about before, perhaps some flags in the TOML that set this or also set "expect route target coordinate" could be part of a user experience improvement for someone down-the-line.